packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -197,6 +208,19 @@ export class FrontendPluginImpl {
       );
     }
 
+    if (isArmSupportEnabled()) {
+      const config = await FrontendConfig.fromPluginContext(ctx);
+      config.endpoint = getArmOutput(ctx, ArmOutput.frontendEndpoint) as string;
```

Comment:
may also need to set the storage account name?

Reply:
storage name is set in FrontendConfig.fromPluginContext.